tests/codebuild/run_integtest.sh

```diff
@@ -45,15 +27,14 @@ function cleanup {
         echo "need_setup_cluster is true, tearing down cluster we created."
         eksctl delete cluster --name "${cluster_name}" --region "${cluster_region}"
         # Delete the role associated with the cluster thats being deleted
-        aws iam detach-role-policy --role-name "${role_name}" --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
-        aws iam delete-role --role-name "${role_name}"
+        delete_generated_role "${role_name}"
+        delete_generated_role "${default_role_name}"
     else
         echo "need_setup_cluster is not true, will remove operator without deleting cluster"
         kustomize build "$path_to_installer/config/default" | kubectl delete -f -
         # Delete the namespaced operator. TODO: This can be cleaner if parameterized
         kustomize build "$path_to_installer/config/crd" | kubectl delete -f -
-        generate_namespace_operator_installer "${crd_namespace}"
-        kubectl delete -f temp_file.yaml    
+        rolebased_operator_installation ${crd_namespace} "config/installers/rolebasedcreds/namespaced" "${role_name}" "delete"  
```

Comment:
Its a bit misnomer. You are calling installation but action can be delete as well. probably just change it to something different `rolebased_operator_operation`

Reply:
Fair, updated.